src/components/AlertModal.jsx

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable class-methods-use-this */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Modal } from 'react-bootstrap';
+import { connect } from 'react-redux';
+import store from '../store/store';
+import { SHOW_ALERT } from '../store/actions/actionTypes';
+
+export class ConnectedAlertModal extends React.Component {
+  constructor() {
+    super();
+
+    this.handleClose = this.handleClose.bind(this);
```

Comment:
@ElMonstro, use arrow functions instead of binding, use of binding brings about performance issues.

Reply:
Arrow functions are hard to test @abulojoshua1 using spyOn